vm-setup/roles/common/defaults/main.yml

```diff
@@ -52,8 +54,8 @@ networks:
     address_v4: "{{ baremetal_network_cidr_v4|nthhost(1)|default('', true) }}"
     netmask_v4: "{{ baremetal_network_cidr_v4|ipaddr('netmask') }}"
     dhcp_range_v4:
-      - "{{ baremetal_network_cidr_v4|nthhost(20) }}"
-      - "{{ baremetal_network_cidr_v4|nthhost(60) }}"
+      - "{{ virsh_dhcp_v4_start }}"
+      - "{{ virsh_dhcp_v4_end }}"
```

Comment:
you do it for v4 but not v6 ?

Reply:
Since we dont use it, I didnt do it, I can do it for v6 as well

Reply:
Fixed